### DISCUSSION TITLE
fabric-textures-v0 not included in version Fabric API v0.45.2 for 1.18.1

### DISCUSSION THREAD

**Discussion by teleheel** [FabricMC/fabric-api#1952] [Player support] [answered]
Running a fresh server with nothing but the latest version of Fabric and the Fabric API produces the warning 

"Mod 'Fabric Rendering Fluids (v1)' (fabric-rendering-fluids-v1) 0.1.18+3ac43d9565 recommends any version of fabric-textures-v0, which is disabled for this environment (client/server only)!
         - You should install any version of fabric-textures-v0 for the optimal experience."

The server boots up anyway and seems to work fine. 

Is this a known issue, or am I missing something?

**Comment by warjort** [comment, marked as answer]
You do have the textures mod, it is a client mod and therefore disabled on the server.
You can ignore this message.

But it shouldn't be telling you to install something that is already installed.
https://github.com/FabricMC/fabric-loader/issues/567

**Reply by teleheel** [reply]
Ah ok, good to know. Thanks!
